// The greeting service definition.
//python3 -m grpc_tools.protoc -I./ --python_out=. --grpc_python_out=. ./backend.proto
syntax = "proto3";
service backendApi {
    rpc sendProductDB (inputMsg) returns (outputMsg) {}
   }
 
message inputMsg {
    optional string input=1;
}   
message outputMsg {
    optional string output = 2;
}

